### PR TITLE
Corrections to the lists of Linux package dependencies

### PR DESCRIPTION
**Pull request by mthalman**
The current set of Linux package dependencies across the supported set of distros have some inaccuracies.  There are cases of packages that aren't listed, packages that shouldn't be listed, and updates to package versions that are needed.

cc @Thraka, @dagood, @MichaelSimons

### CODE REVIEW

**Comment by dagood** [comment]
+cc @NikolaMilosavljevic

**Comment by Thraka** [comment]
I was just having a conversation about this with @leecow 

I just need the lists corrected by someone with the knowledge and I can get it updated.

File: docs/core/install/dependencies.md [hunk 1]
```diff
@@ -241,16 +241,17 @@ Based on your linux distribution, you may need to install additional dependencie
 
 Ubuntu distributions require the following libraries to be installed:
 
-- liblttng-ust0
-- libcurl3 (for 14.x and 16.x)
-- libcurl4 (for 18.x)
-- libssl1.0.0
-- libkrb5-3
-- zlib1g
+- libc6
```
Comment (leecow) [inline docs/core/install/dependencies.md:244]: Where did you get the requirements for libc6 and libgcc1? I have not seen a need to install these independently for .NET scenarios.

File: docs/core/install/linux-debian.md [hunk 2]
```diff
@@ -122,7 +122,7 @@ sudo apt-get update; \
 
 ## Dependencies
 
-[!INCLUDE [linux-install-dependencies](includes/linux-install-dependencies.md)]
+[!INCLUDE [linux-rpm-install-dependencies](includes/linux-rpm-install-dependencies.md)]
```
Comment (leecow) [inline docs/core/install/linux-debian.md:125]: The dependency list for Debian is not the same as what is listed in the rpm document. 

Debian 10: libicu63, libssl1.1
Debian 9: libicu57, libssl1.1

**Review by leecow — CHANGES_REQUESTED** [review]
Left a few comments for changes and a question.

File: docs/core/install/dependencies.md (hunk 1 above)
Comment (mthalman) [reply docs/core/install/dependencies.md:244]: From here:
* https://github.com/dotnet/runtime/blob/a755e742cad93fb173b47a95f68168c29ae95fb6/src/installer/pkg/packaging/deb/dotnet-runtime-deps-debian_config.json#L32
* https://github.com/dotnet/dotnet-docker/blob/75e441fdb26bdffa5f58437cd77048586016ec96/src/runtime-deps/3.1/bionic/amd64/Dockerfile#L8

This was before my time so I don't know the history.  Perhaps @dagood or @MichaelSimons has more context.

Comment (dagood) [reply docs/core/install/dependencies.md:244]: As far as Git can tell me, these originated in https://github.com/dotnet/core-setup/pull/3317.

File: docs/core/install/linux-debian.md (hunk 2 above)
Comment (mthalman) [reply docs/core/install/linux-debian.md]: Fixed

**Review by leecow — APPROVED** [review]
lgtm

**Pull request merged**